WhateverGreen/kern_igfx.cpp

```diff
@@ -761,6 +765,9 @@ bool IGFX::wrapAcceleratorStart(IOService *that, IOService *provider) {
 
 		}
 	}
+	
+	OSObject *metalPluginName = that->getProperty("MetalPluginName");
```

Comment:
I guess you should dynamic-cast here, as doing it later may result in null pointer dereference. Also, please check the CI failure.

Reply:
`getProperty` returns an `OSObject *` so I think it's fine. But I should add `retain` and `release` because it's possible that `start` will dealloc the property.

Reply:
The issue was that you dunamic casted a pointer and immediately dereferenced it by getting C string without checking for NULL. The latest change resolved the problem, but made clang static analyzer unhappy: https://travis-ci.com/github/acidanthera/WhateverGreen/jobs/350020803

Could you please fix it up?

Reply:
I think the static analyzer is too stupid to recognize that callbackIGFX->forceMetal cannot change between calls. I can try to always retain and release it but it might look ugly.